### PR TITLE
LRDOCS-8469 - Forms Configuration Reference

### PR DESCRIPTION
**Pull request by justinchoi001**
@rbohl

### CODE REVIEW

**Comment by rbohl** [comment]
@sez11a @jrhoun Justin created a stub for _Importing and Exporting Forms_ (LAR import/export). Is this something we want docs for on Liferay Learn for each asset that implements import/export? The implementation details can be somewhat different for each asset type, but I wasn't sure if we will just cover import/export generally as a separate topic or if we'll do it app-by-app/asset-by-asset. Just figured we should bubble this up to you guys.

File: docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+| From Name | The sender's name; this could be the Site name, the form name, or anything else informative to the recipient. |
+| From Address | The sender's email address; You can use `noreply@example.com` to prevent recipients from replying. |
+| To Address | The recipient's email address (e.g., `test@example.com`) |
+| Subject | An informative subject line tells the recipient what happened. |
+
+To learn more, see [Configuring Form Notifications](./configuring-form-notifications.md).
+
+## Form Widget Configuration
+
+You can configure the _Form_ widget deployed to a Site Page. To access the _Configuration_ menu, click the ![Options](../../../images/icon-options.png) icon next to the widget title then _Configuration_.
```
Comment (rbohl) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md:49]: Minor quibble. I think this should refer to `icon-app-options.png`

Comment (justinchoi001) [reply docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md:49]: Switched icons. Thanks.

File: docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md [hunk 2]
```diff
@@ -1,3 +1,72 @@
 # Forms Configuration Reference
 
-Coming soon!
+This article serves as a reference for all Forms and the Form application configurations.
```
Comment (rbohl) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md:3]: I think this violates the [meta-information](https://docs.google.com/document/d/1RxAwUhr4-JKpIEp1nriOL4GRctwCH3GQ7Ng998s4EHU/edit#heading=h.d9h1av2r0poy) rule in the standards. I don't mind it so much in a configuration reference article, but Rich may disagree more strongly.

One way to give it more meaning would be to contrast it with the feature documentation:

>  Most of the time you can find documentation for Forms configuration within the articles documenting Forms features. By contrast, this article provides an overview of all the Forms configurations so you can quickly find a configuration you need.

Maybe @sez11a has another take on this.

**Comment by rbohl** [comment]
@justinchoi001 We probably should add the System and Instance Settings entries for Forms as well. Screenshots attached.

forms-inst-settings
forms-sys-settings

File: docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md (hunk 2 above)
Comment (sez11a) [reply docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md:3]: It does. We don't say "this article" does this or that or anything.

Comment (justinchoi001) [reply docs/dxp/7.x/en/process-automation/forms/user-guide/forms-configuration-reference.md]: Edited

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/816

**Pull request closed without merging**